File: src/main/java/net/nextpulse/jacumulus/models/PaymentStatusEnum.java
package net.nextpulse.jacumulus.models;

/**
 * Enum representing the payment status of an invoice.
 */
public enum PaymentStatusEnum {
  Due(1),
  Paid(2);
  
  private final int value;
  
  PaymentStatusEnum(int i) {
    this.value = i;
  }
  
  public String toString() {
    return String.valueOf(value);
  }
}

File: src/main/java/net/nextpulse/jacumulus/models/PaymentStatusEnum.kt
package net.nextpulse.jacumulus.models

/**
 * Enum representing the payment status of an invoice.
 */
enum class PaymentStatusEnum constructor(private val value: Int) {
    Due(1),
    Paid(2);

    override fun toString(): String {
        return value.toString()
    }
}
